nistats/datasets.py

```diff
@@ -55,33 +58,37 @@ def fetch_bids_langloc_dataset(data_dir=None, verbose=1):
     return os.path.join(data_dir, main_folder), sorted(file_list)
 
 
-def fetch_openfmri_dataset(dataset_name='ds000001', dataset_revision=None,
-                           data_dir=None, verbose=1):
-    """Download latest revision of specified bids dataset.
-
-    Compressed files will not be uncompressed automatically due to the expected
-    great size of downloaded dataset.
-
-    Only datasets that contain preprocessed files following the official
-    conventions of the future BIDS derivatives specification can be used out
-    of the box with Nistats. Otherwise custom preprocessing would need to be
-    performed, optionally following the BIDS derivatives specification for the
-    preprocessing output files.
+def fetch_openneuro_dataset(data_dir=None, dataset_version='ds000030_R1.0.4',
+                            n_subjects=1, inclusion_filters=[],
+                            exclusion_filters=[], verbose=1):
+    """Download language localizer example bids dataset.
 
     Parameters
     ----------
-    dataset_name: string, optional
-        Accesion number as published in https://openfmri.org/dataset/.
-        Downloads by default dataset ds000001.
-
-    dataset_revision: string, optional
-        Revision as presented in the specific dataset link accesible
-        from https://openfmri.org/dataset/. Looks for the latest by default.
-
     data_dir: string, optional
         Path to store the downloaded dataset. if None employ nilearn
         datasets default download directory.
 
+    dataset_version: string, optional
+        dataset version name. Assumes it is of the form [name]_[version].
+
+    n_subjects: int, optional
+        number of subjects to download from the dataset.
+
+    inclusion_filters: list of str, optional
+        List of regex strings that will be used to filter the url list.
+        If the regex matches the url it is retained for download.
+        Multiple filters work on top of each other.
+        Like an "and" logical operator, creating a more restrictive query.
+        Inclusion and exclusion filters apply together.
```

Comment:
This is typically a case where the API becomes daunting to users without a very explicit example. Please provide one.